mongodbatlas/fw_data_source_mongodbatlas_project.go

```diff
@@ -0,0 +1,255 @@
+package mongodbatlas
+
```

Comment:
does "fw_" ahead of each of the file names stand for "framework"? if so suggest we call out somewhere in repo README file to let users know which resources / data sources have been migrated to new Terraform Plugin Framework

Reply:
@Zuhairahmed added to CONTRIBUTING.md